### PR TITLE
Updates for 15.0 release  Closes #1063 and #1077

### PR DESCRIPTION
**Pull request by Dude4Linux**
@JedMeister - I've been holding off issuing this PR hoping to find time to work on the apt proxy issue that we discussed earlier.  Since that will involve changes to common that will affect all apps, I'm guessing you will want to wait for a future release.  I've tested this latest version in a ProxMox VM and in an LXC v2 container.  The ProxMox VM works as expected, but unfortunately the container install fails, even with security.nesting and security.privileged enabled.  I thought I had this working for the 14.2 version of the LXC appliance.  Will have to do some further checking to be sure.

Closes #1063 and #1077

### CODE REVIEW

**Comment by Dude4Linux** [comment]
@JedMeister @OnGle - I restored `python-lxc` to `plan/main` after testing showed it was still required by Ansible.

**Comment by JedMeister** [comment]
Great thanks @Dude4Linux

File: overlay/usr/share/lxc/templates/lxc-turnkey [hunk 1]
```diff
@@ -321,13 +310,17 @@ configure_turnkey() {
     # ensure inithooks.conf is executable by root
     chmod 0700 ${rootfs}/etc/inithooks.conf
 
+    # remove the ntp servers; containers get time from host
+    info "disabling ntp servers..."
+    sudo sed -i '/^NTPSERVERS=/ s/".*"/""/' ${rootfs}/etc/default/ntpdate
```
Comment (JedMeister) [inline overlay/usr/share/lxc/templates/lxc-turnkey]: Should this sudo be here? I'm guessing not?

**Comment by JedMeister** [comment]
Other than the stray `sudo` I noticed, this looks fantastic @Dude4Linux. Let me know about that and I'll merge ASAP.

**Comment by Dude4Linux** [comment]
@JedMeister - Nice catch!! That's what happens when you test in an unprivileged Ubuntu account, then cut and paste the code.  Funny I never noticed the warning messages. I'm sure that's not the only thing I missed.

**Comment by JedMeister** [comment]
@Dude4Linux - I haven't done final testing, but I did do another full review of the code and it looks good so I'm going to merge. If you find or think of anything else, just add a new PR. Thanks again John. :smile:

**Comment by Dude4Linux** [comment]
@JedMeister @OnGle - Using Ansible (v14.2) and LXC v15.0rc1, I ran a playbook to create and test all 100 v14.2 appliances.  The only failures were prestashop and zurmo, both of which exhibited the redirect to www.example.com problem that I reported earlier.  I presume that both have been fixed in the 15.0 release. I don't have any way to test v15.0 proxmox images, since none have been released yet (AFAIK).

One issue with the LXC v15.0rc1 appliance that I noticed is that the `apt-cacher-ng` is installed but is not configured so that `apt caching` is not working as desired.  I'm looking at ways to get the containers to use the LXC host's `apt cache`.  I think the best way to proceed is to add `APT_PROXY` to `inithooks.conf` and add `10proxy` to `firstboot.d` to capture `APT_PROXY` and configure `/etc/apt/apt.conf.c/01proxy`.

I'm also considering the merits of adding Polipo and configuring it to handle HTTP requests from `wget`, `curl`, and `git`.  Not much can be done with HTTPS at the moment.  

Since this branch has been merged, I've closed it and started a new branch.

**Comment by JedMeister** [comment]
> One issue with the LXC v15.0rc1 appliance that I noticed is that the `apt-cacher-ng` is installed but is not configured so that apt caching is not working as desired. I'm looking at ways to get the containers to use the LXC host's apt cache. I think the best way to proceed is to add `APT_PROXY` to `inithooks.conf` and add `10proxy` to `firstboot.d` to capture `APT_PROXY` and configure `/etc/apt/apt.conf.c/01proxy`.

That sounds like a nice plan and I like it. Were you thinking of implementing that within the LXC appliance itself? I have no issue with that, at least initially. 

Ultimately, it should possibly be included in the [`inithooks`](https://github.com/turnkeylinux/inithooks) repo (disabled by default). 

As a somewhat tangential point, ideally I think that all the inithooks should probably be consolidated into the inithooks source package. We could then provide a minimal `inithooks-common` binary package which includes only the commonly shared ones and additional packages (which depend on `inithooks-common`) to provide additional inithooks.

Having said that, that's something for another day... The v15.0 release is already so far behind schedule I'm not even going to consider anything as radical as that right now! :smile:

**Comment by Dude4Linux** [comment]
Moving this discussion to #1083 as this is now closed.

**Pull request merged**